src/models/brands.ts

```diff
@@ -0,0 +1,19 @@
+import { DataTypes } from 'sequelize';
+import { sequelize } from "../config/database";
+export const BrandsModel = sequelize.define('Brand', {
+    id: {
+        type: DataTypes.INTEGER,
+        autoIncrement: true,
+        primaryKey: true,
+      },
+      name: {
+        type:DataTypes.STRING(40),
+        allowNull:false,
+      },
+      image_secure_url:{
```

Comment:
Why did you add "secure" ? is the url encrypted for some reason ?

Reply:
No, I kept it in Mohammad's way because he named it that way in other models.

Reply:
secure_url as it named by the Cloudinary, its a link to the image in the fileSystem.